pkg/crypto/measure.go

```diff
@@ -0,0 +1,59 @@
+package crypto
+
+import (
+	"io/ioutil"
+	"log"
+
+	"github.com/systemboot/tpmtool/pkg/tpm"
+)
+
+const (
+	// Blob type in PCR 7
+	Blob uint32 = 7
+	// BootConfig type in PCR 8
+	BootConfig uint32 = 8
+	// ConfigData type in PCR 8
+	ConfigData uint32 = 8
+	// NvramVars type in PCR 9
+	NvramVars uint32 = 9
+)
+
+// TryMeasureBootConfig measures bootconfig contents
+func TryMeasureBootConfig(name string, kernel string, initramfs string, kernelArgs string, deviceTree string) {
```

Comment:
nit: Wouldn't it be easier if this function was in bootconfig instead compared to having to unpack the struct to use it ? everything it uses is exported

Reply:
I agree with you, this would probably better live in the bootconfig package